ui/app/appointments/controllers/manage/list/appointmentsListViewController.js

```diff
@@ -177,6 +186,7 @@ angular.module('bahmni.appointments')
                     $scope.selectedAppointment.status = response.data.status;
                     closeConfirmBox();
                     messagingService.showMessage('info', message);
+                    $scope.getAppointmentsForDateNew($stateParams.viewDate);
```

Comment:
Instead of getting new appointments, you can just update the scope variable filteredAppointments